src/Persistence/Initialization/BaseMapInitializer.cs

```diff
@@ -127,6 +127,18 @@ public void SetSafezoneMap()
         }
 
         var safezoneMap = this.GameConfiguration.Maps.FirstOrDefault(map => map.Number == this.SafezoneMapNumber);
+        switch (this._mapDefinition.GetName())
```

Comment:
Well, it will work, but there is a cleaner approach.
Just override the SafezoneMapNumber property in the Icarus and Karutan2 classes ;-)

Reply:
thx will do that instead makes more sense xD.

Didnt realize that there is specific map classes yet xD